Kitodo-FileManagement/src/test/java/org/kitodo/filemanagement/FileManagementTest.java

```diff
@@ -191,6 +191,25 @@ public void shouldDeleteDirectory() throws IOException {
     }
 
     @Test
+    public void shouldGetSizeOfDirectory() throws Exception {
+        int testContent = 156575;
+
+        URI resource = fileManagement.create(URI.create("fileTest"), "size.txt", true);
+        Assert.assertTrue(fileManagement.fileExist(resource));
+
+        OutputStream outputStream = fileManagement.write(URI.create("fileTest/size.txt"));
```

Comment:
Please use try-with-resources statement for the `outputStream` object to avoid possible resources leaks.